mis_builder/migrations/8.0.2.0.0/post-migration.py

```diff
@@ -2,6 +2,27 @@
 # © 2017 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
+from openerp import api, SUPERUSER_ID
+
+
+def fill_defaults(env):
+    cr = env.cr
+
+    default_specs = {
+        'mis.report.instance.period': ['report_instance_id'],
+        'mis.report.kpi': ['report_id'],
+        'mis.report.query': ['report_id'],
```

Comment:
`mis.report.instance.period` without report instance make no sense. Same for kpi and queries without reports. If you have those in your 8.0 database, you should simply drop them. They are the result of bugs (missing unlink or cascade delete).